doc/architecture/decisions/0002-object-identifiers.md

```diff
@@ -0,0 +1,57 @@
+# 0002. Object identifiers
+
+Date: 2019-06-11
+
+## Status
+
+Agreed
+
+## Context
+
+To uniquely identify an object in Ensembl we need three pieces of information:
+
+-   The **dataset** to which the object belongs, e.g. homo_sapiens_GCA_000001405_14 [^1]
```

Comment:
This is the assembly an object belongs to. But how do we differentiate a situation like an unchanged identifier between annotation runs e.g. BRCA2.

- `ENSG00000139618.15` is the current version in Ensembl release 97 (GENCODE31)
- `ENSG00000139618.14` is live in Ensembl releases 96 and 95 (GENCODE30)
- `ENSG00000139618.14` is live in Ensembl releases 94 and 93 (GENCODE29)

`assembly_id` alone is insufficient to designate what this dataset is. I'd recommend switching this to state that `assembly_id` could be the start of an identifier but equally applicable is using `GENCODE30` so long as it is unique.